platform/openide.dialogs/src/org/openide/NotifyDescriptor.java

```diff
@@ -129,24 +129,45 @@ public class NotifyDescriptor extends Object {
      */
     public static final String PROP_INFO_NOTIFICATION = "infoNotification"; // NOI18N
 
+    /**
```

Comment:
Does this worth doing at all? The only advantage over simple auto boxing is to have unique Object instances for each option.

Do we really need that? Also it could break comparisons like `ret.equals(0)`.

```java
Object YES_OPTION = JOptionPane.YES_OPTION;
```

would make more sense.

Reply:
this doesn't work since two options would have the same value and same identity which would be an incompatible change.

Reply:
moved the `NotifyDescriptor` changes to a separate PR https://github.com/apache/netbeans/pull/8407